hc/api/management/commands/ensuretriggers.py

```diff
@@ -9,6 +9,7 @@ def _pg(cursor):
         BEGIN
             IF NEW.last_ping IS NOT NULL THEN
                 NEW.alert_after := NEW.last_ping + NEW.timeout + NEW.grace;
+                NEW.nag_after := CURRENT_TIMESTAMP + NEW.nag_interval;
```

Comment:
Why do we need this trigger right now?

Reply:
When the Nag mode is updated for a check that has been down for hours, if we don't trigger, the nag_after won't be updated.

Reply:
I will remove this and let it update when the check goes down, as discussed.